app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= n_('Displaying %{num_audits} of %{total_audits} audits',
+         'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
+  <% @audits.each do |audit| %>
```

Comment:
Indentation for conditionals and for loops should be removed in this mailer, because it's text it all shows up.